### PR TITLE
feat(backend): Renaming Aio and Null to Volume

### PR DESCRIPTION
**Pull request by glimchb**
This brings more consistency to the naming of backend

See opiproject/opi-api#304

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Comment by glimchb** [comment]
See https://github.com/opiproject/opi-api/issues/304

File: pkg/backend/null.go [hunk 1]
```diff
@@ -77,15 +77,15 @@ func (s *Server) CreateNullDebug(_ context.Context, in *pb.CreateNullDebugReques
...
+	response := server.ProtoClone(in.NullVolume)
+	s.Volumes.NullVolumes[in.NullVolume.Name] = response
+	log.Printf("CreateNullVolume: Sending to client: %v", response)
 	return response, nil
 }
 
-// DeleteNullDebug deletes a Null Debug instance
-func (s *Server) DeleteNullDebug(_ context.Context, in *pb.DeleteNullDebugRequest) (*emptypb.Empty, error) {
-	log.Printf("DeleteNullDebug: Received from client: %v", in)
+// DeleteNullVolume deletes a Null Debug instance
```
Comment (artek-koltun) [inline pkg/backend/null.go:86]: deletes a Null volume

File: pkg/backend/null.go [hunk 2]
```diff
@@ -286,12 +286,12 @@ func (s *Server) GetNullDebug(_ context.Context, in *pb.GetNullDebugRequest) (*p
...
 		return nil, status.Errorf(codes.InvalidArgument, msg)
 	}
-	return &pb.NullDebug{Name: result[0].Name, Uuid: &pc.Uuid{Value: result[0].UUID}, BlockSize: result[0].BlockSize, BlocksCount: result[0].NumBlocks}, nil
+	return &pb.NullVolume{Name: result[0].Name, Uuid: &pc.Uuid{Value: result[0].UUID}, BlockSize: result[0].BlockSize, BlocksCount: result[0].NumBlocks}, nil
 }
 
-// NullDebugStats gets a Null Debug instance stats
-func (s *Server) NullDebugStats(_ context.Context, in *pb.NullDebugStatsRequest) (*pb.NullDebugStatsResponse, error) {
-	log.Printf("NullDebugStats: Received from client: %v", in)
+// NullVolumeStats gets a Null Debug instance stats
```
Comment (artek-koltun) [inline pkg/backend/null.go:292]: gets a Null volume stats

File: pkg/backend/null.go [hunk 3]
```diff
@@ -241,18 +241,18 @@ func (s *Server) ListNullDebugs(_ context.Context, in *pb.ListNullDebugsRequest)
...
-	sortNullDebugs(Blobarray)
-	return &pb.ListNullDebugsResponse{NullDebugs: Blobarray, NextPageToken: token}, nil
+	sortNullVolumes(Blobarray)
+	return &pb.ListNullVolumesResponse{NullVolumes: Blobarray, NextPageToken: token}, nil
 }
 
-// GetNullDebug gets a a Null Debug instance
-func (s *Server) GetNullDebug(_ context.Context, in *pb.GetNullDebugRequest) (*pb.NullDebug, error) {
-	log.Printf("GetNullDebug: Received from client: %v", in)
+// GetNullVolume gets a a Null Debug instance
```
Comment (artek-koltun) [inline pkg/backend/null.go:253]: gets a a Null volume

File: pkg/backend/null.go [hunk 4]
```diff
@@ -126,26 +126,26 @@ func (s *Server) DeleteNullDebug(_ context.Context, in *pb.DeleteNullDebugReques
 	return &emptypb.Empty{}, nil
 }
 
-// UpdateNullDebug updates a Null Debug instance
-func (s *Server) UpdateNullDebug(_ context.Context, in *pb.UpdateNullDebugRequest) (*pb.NullDebug, error) {
-	log.Printf("UpdateNullDebug: Received from client: %v", in)
+// UpdateNullVolume updates a Null Debug instance
```
Comment (artek-koltun) [inline pkg/backend/null.go:129]: updates a Null volume

**Review by artek-koltun — CHANGES_REQUESTED** [review]

**Review by artek-koltun — APPROVED** [review]
LGTM

**Pull request merged**